assets/css/style.css

```diff
@@ -0,0 +1,197 @@
+body {
+  max-width: 1310px;
+  font-family: 'Roboto Condensed', Arial, Helvetica, sans-serif;
+  background-color: #f4f5f6;
+  margin: 0 auto;
+}
+
+main,
+footer {
+  background-color: white;
+}
+
+h3 {
+  font-size: 1.4rem;
+  font-weight: bold;
+  line-height: 1.3
+}
+
+h3>a,
+h3>a:link,
+h3>a:visited,
+h4>a,
+h4>a:link,
+h4>a:visited {
+  color: black;
+}
+
+h3>a:hover,
+h3>a:active,
+h4>a:hover,
+h4>a:active {
+  text-decoration: none;
+  border-bottom: 2px solid #f72210
+}
+
+.card-link-custom:hover,
+.card-link-custom:active {
+  color: #f72210;
+  border: none
+}
+
+h4 {
+  font-size: 1.1rem;
+  font-weight: bold;
+  line-height: 1.2
+}
+
+.card-text-custom-bold {
+  color: #555;
+  font-weight: 700;
+  line-height: 1.2;
+}
+
+.make-uppercase {
+  text-transform: uppercase;
+  color: rgba(3, 3, 3, .4)
+}
+
+html {
+  font-size: 1rem;
+}
+
+/* MAGAZINES SECTION */
+
+/* container header */
+.container-header {
+  display: flex;
+  justify-content: center;
+}
+
+.container-header>* {
+  align-self: center;
+  margin: 0;
+}
+
+.nw-icon {
+  background: url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain no-repeat #f72210;
+  width: 29px;
+  height: 28px;
+  margin-right: 10px;
+}
+
+/* overlay text */
+.overlay-relative {
+  position: relative;
+}
+
+.overlay-text {
+  display: inline-block;
+  position: absolute;
```

Comment:
`SUGGESTION`

You can also use the `d-inline-block` and  `position-absolute` class name for this selector element and in all other occurrences.